README.md

```diff
@@ -70,6 +70,19 @@ In alphabetic order.
     </td>
   </tr>
 
+  <tr>
+    <td>
+      <img width="300px" src=""/>
+    </td>
+    <td>
+       <h1>Gensim</h1>
+       <p>
+       </p>
```

Comment:
"Topic Modelling for Humans". Gensim is an open-source Python library for topic modelling, document indexing and similarity retrieval with large corpora. Target audience is the natural language processing (NLP) and information retrieval (IR) community. Memory-independent and distributed algorithms.